javascript/javascript1/week1/main.js

```diff
@@ -0,0 +1,51 @@
+// Age-ify (A future age calculator)
+let yearOfBirth = 1978;
+let yearFuture = 2045;
+let age = yearFuture - yearOfBirth;
+console.log ('You will be ' + age + ' years old in ' + yearFuture + ' ');
+
+//Goodboy-Oldboy (A dog age calculator)
+let dogYearOfBirth = 2010;
+let dogYearFuture = 2025;
+let dogYear = 7;
+// assume that 1 dog year equals to 5 human years
+let shouldShowResultInDogYears = false;
+if(shouldShowResultInDogYears === true) {
+let ageOfDog = (dogYearFuture - dogYearBirth) * dogYear;   
```

Comment:
Does not look like dogYearBirth is defined. I get Uncaught ReferenceError: dogYearBirth is not defined, if i change the  shouldShowResultInDogYears  to true. Also look into const vs let